src/pint/models/timing_model.py

```diff
@@ -55,8 +57,24 @@
         #    'NE_SW', 'NE_SW2',
     ]
 )
+
 ignore_prefix = set(["DMXF1_", "DMXF2_", "DMXEP_"])  # DMXEP_ for now.
 
+DEFAULT_ORDER = [
```

Comment:
Two questions:  Why is jump_delay *after* astrometry?  Isn't a jump_delay a time jump and so really means change our input times by this much?  That seems like that should be applied first before everything else.   Second, why is jump_delay different from phase_jump?  Shouldn't they both have the same form?  Maybe time_jump or delay_jump would be better if we like phase_jump?  Or, alternately, maybe phase_jump should be jump_phase....  I think I might like that one best.